src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,53 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.Comparator;
+
+public enum LottoRank {
+    FIRST_PLACE(new Integer(6), 2000000000),
```

Comment:
new Integer를 사용하신 이유가 있을까요?

또한 언더바를 통해 가독성을 상승시킬 수 있습니다!

```suggestion
    FIRST_PLACE(new Integer(6), 2_000_000_000),
```

Reply:
이 부분은 사실.. 이제와서 new Integer은 사용할 필요가 없다는 것을 깨달았습니다..
Map에서 2등과 3등을 구분 할 때 버그가 있어서, 저 부분에 new Integer을 끼워넣는 것으로 해결 하려 했으나
사실 SECOND_PLACE와 THIRD_PLACE 는 다른 주소값을 가지기 때문에 new Integer을 넣는 것은 아무 의미가 없었네요